admin/data/sql/migrations/113.do.add-access-arrangements-id-field.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE [mtc_admin].pupilAccessArrangements ADD accessArrangements_id int NOT NULL
+ALTER TABLE [mtc_admin].pupilAccessArrangements
+ADD CONSTRAINT FK_pupilAccessArrangements__accessArrangements_id
```

Comment:
Is this a double-underscore - in which case is this a typo?

Reply:
@jon-shipley yes very good catch. Webstorm generated it this way. I am updating this.

Reply:
@jon-shipley unfortunately this is now merged since it was part of #787.
I'll generate an additional migration which renames this one.